plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseTestHiveOnDataLake.java

```diff
@@ -383,14 +383,24 @@ public void testEnumPartitionProjectionOnVarcharColumnWithWhitespace()
     @Test
     public void testEnumPartitionProjectionOnVarcharColumnWithStorageLocationTemplateCreatedOnTrino()
     {
+        // It's important to mix case here to detect if we properly handle rewriting
+        // properties between Trino and Hive (e.g for Partition Projection)
+        String schemaName = "Hive_Datalake_MixedCase";
         String tableName = getRandomTestTableName();
+
+        // We create new schema to include mixed case location path and create such keys in Object Store
+        computeActual(format(
```

Comment:
No need to wrap here. Also, `formatted()` is preferred for new code:
```java
computeActual("CREATE SCHEMA hive.%1$s WITH (location='s3a://%2$s/%1$s')".formatted(schemaName, bucketName));

Reply:
Applied